docs/src/documentation-pages/dev/red.md

```diff
@@ -0,0 +1,25 @@
+The **red** deployment setup is meant for environments in which public access is not available (such as the [COOL](https://github.com/cisagov/cool-system)). In this case, we do not use
+API Gateway or Cloudfront to handle the backend / frontend. Rather, each user needs to spin up a lightweight instance of
+the frontend / backend on their AWS machine. Note that all the scanning and accessing the database is still done via AWS lambda / AWS ECS.
+
+The user builds and runs the following Docker containers on their AWS machine:
```

Comment:
FYI @cisagov/team-ois this is the alternative to using API Gateway / Cloudfront that I mentioned in my email